validator/README.md

```diff
@@ -0,0 +1,52 @@
+## OpenTelemetry SDK Configuration Validator
+
+This application will validate a yaml or json file against the [OpenTelemetry
+SDK Configuration schema](https://github.com/open-telemetry/opentelemetry-configuration/).
+
+### Build
+
+The `schema` directory is required to be in the directory of the Go file that
+embeds it so a `go build` alone will fail, instead run `make` which will copy
+the schema:
+
+```
+$ make
+```
+
+Same is true for building the docker image:
+
+```
+$ make docker
```

Comment:
Can you provide examples demonstrating usage of the docker image?

Reply:
Will do.

Reply:
Added.